src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
So looking at the [debian policy 9.9](https://www.debian.org/doc/debian-policy/ch-opersys.html#s9.9) closely it's actually pretty clear about how to set things. It says: 
`If a program usually depends on environment variables for its configuration, the program should be changed to fall back to a reasonable default configuration if these environment variables are not present.`

And it gives an example: 
```
     #!/bin/sh
     BAR=${BAR:-/var/lib/fubar}
     export BAR
     exec /usr/lib/foo/foo "$@"
```
Which uses the syntax `:-` which only sets the variable if it's unset. [description here](https://unix.stackexchange.com/questions/122845/using-a-b-for-variable-assignment-in-scripts)

The sentence does not say "if and only if", but the example explicitly uses that logic.